etc/openqa/openqa.ini

```diff
@@ -161,6 +161,7 @@ blocklist = job_grab job_done
 #untracked_assets_storage_duration = 14
 # Performs the cleanup of results/assets only if the free disk space on the relevant partition is below the specified percentage (and aborts early otherwise)
 #result_cleanup_max_free_percentage = 100
+#trigger_cleanup_on_job_done = 0
```

Comment:
This should be in the middle of `result_cleanup_max_free_percentage` and `asset_cleanup_max_free_percentage`. It would make sense to put `trigger_cleanup_on_job_done` under its own explaining comment.